lib/appoptics_apm/inst/graphql.rb

```diff
@@ -112,20 +124,22 @@ def span_name(key)
 
           # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
           def metadata(data, layer)
-            data.keys.map do |key|
+            kvs = data.keys.map do |key|
               case key
               when :context
-                graphql_context(data[key], layer)
+                graphql_context(data[:context], layer)
```

Comment:
I changed it because it was a pain when debugging
why do you think it is better?

Reply:
Because the `key` is `:context` here, I think it is unnecessary to reference `:context` explicitly

Reply:
that's true
it is not for functionality that I changed it is because it is easier to see what is going on, especially when debugging and the top of the case statement is not visible, so I would like to leave it for now.

Reply:
okay. Got it.